test/browser_launchers/playwright_launcher.test.js

```diff
@@ -156,4 +158,66 @@ describe('Apify.launchPlaywright()', () => {
             if (browser) await browser.close();
         }
     });
+
+    describe('Default browser path', () => {
+        const path = 'test';
+
+        beforeAll(() => {
+            process.env.APIFY_DEFAULT_BROWSER_PATH = path;
+        });
+
+        afterAll(() => {
+            delete process.env.APIFY_DEFAULT_BROWSER_PATH;
+        });
+
+        test('uses Apify default browser path', () => {
+            const launcher = new PlaywrightLauncher({
+                launcher: {},
+            });
+            const plugin = launcher.createBrowserPlugin();
+
+            expect(plugin.launchOptions.executablePath).toEqual(path);
+        });
+        test('does not use default when using chrome', () => {
+            const launcher = new PlaywrightLauncher({
+                useChrome: true,
+                launcher: {},
+            });
+            const plugin = launcher.createBrowserPlugin();
+
+            expect(plugin.launchOptions.executablePath).not.toBe(path);
```

Comment:
I don't know if this is a good test. It tests that the env var is not used, but it does not test whether the path is not broken in some other way. What is the expected behavior? That this will be set to `undefined`? Then we should test for `undefined`

Reply:
Good point 👍